### PR TITLE
Enable using tls in client cli

### PR DESCRIPTION
**Pull request by Tibo-lg**
As the title describes, this PR enables passing a path to the server certificate to communicate with it through TLS.

### CODE REVIEW

File: cmd/p2pdcli/p2pdcli.go [hunk 1]
```diff
@@ -92,14 +95,24 @@ func main() {
...
-	opts = append(opts, grpc.WithInsecure())
+	if *serverCertificate == "" {
+		opts = append(opts, grpc.WithInsecure())
+		return grpc.Dial(*serverAddress, opts...)
+	}
+
+	creds, err := credentials.NewClientTLSFromFile(*serverCertificate, "")
+
+	if err != nil {
+		panic(err)
```
Comment (prunelsigma) [inline cmd/p2pdcli/p2pdcli.go]: maybe return the error as you are handling the error (add message and log) on return

**Review by prunelsigma — CHANGES_REQUESTED** [review]
just one comment :)

File: cmd/p2pdcli/p2pdcli.go (hunk 1 above)
Comment (Tibo-lg) [reply cmd/p2pdcli/p2pdcli.go]: Good point done.

**Review by prunelsigma — APPROVED** [review]
LGTM

**Pull request merged**